devices/pine64-pinephone/sound.nix

```diff
@@ -0,0 +1,22 @@
+# PinePhone UCM files
+#
+# A hack to avoid mass rebuilds of packages that depend on
+# alsaLib.
+
+{ config, lib, ... }:
+
+let
+  ucm2 = "${./ucm2}";
```

Comment:
Please add a comment describing the origin of the files, so someone in the future can follow them up for a possible update.